avail-subxt/examples/vector_send_msg.rs

```diff
@@ -0,0 +1,182 @@
+use avail_core::{
+	data_proof::{tx_uid, AddressedMessage, BoundedData, Message, SubTrie},
+	AppId, Keccak256,
+};
+use avail_subxt::{
+	api, avail_client::RpcMethods, rpc::KateRpcClient as _, tx, AccountId, AvailClient, Opts,
+};
+
+use binary_merkle_tree::verify_proof;
+use derive_more::Constructor;
+use futures::stream::{FuturesOrdered, TryStreamExt as _};
+use hex_literal::hex;
+use std::collections::HashSet;
+use structopt::StructOpt;
+use subxt::{backend::BlockRef, error::RpcError, utils::H256, Error};
+use subxt_signer::sr25519::dev;
+
+const DATA: &[u8] = b"Test 42";
+const DOMAIN: u32 = 2;
+
+fn messages() -> Vec<Message> {
+	let asset_id = hex!("4554480000000000000000000000000000000000000000000000000000000000");
```

Comment:
Transaction with this asset id must fail.
`SUPPORTED_ASSET_ID` is only Avail H256::zero()

Reply:
I've just used the **same asset id from previous version** of test `avail-subxt/examples/bridge_send_message.rs` at line 23. It was success on previous version and it still works.
Pls keep in mind this is executed using the `--dev` chainspec.

Should this fail in `dev` chainspec?

Reply:
> Should this fail in dev chainspec?
yes, because the supported asset is only Avail which is `0x0..0`.

Reply:
The check got removed in the sequence recovery PR, and we somehow missed it to check. I will add it back.